js/src/components/modals/PostSettingsModal.js

```diff
@@ -2,6 +2,107 @@ import { Button } from "@yoast/components/src/button/Button";
 import { __, sprintf } from "@wordpress/i18n";
 import { useState, useCallback } from "@wordpress/element";
 import Modal from "./Modal";
+import Collapsible from "../SidebarCollapsible";
+import SnippetEditor from "../../containers/SnippetEditor";
+import FacebookContainer from "../../containers/FacebookEditor";
+import TwitterContainer from "../../containers/TwitterEditor";
+import SchemaTabContainer from "../../containers/SchemaTab";
+import AdvancedSettings from "../AdvancedSettings";
+import PropTypes from "prop-types";
+
+/**
+ * Returns the ModalContent.
+ *
+ * We need this in a function since the scripData is not available earlier.
+ *
+ * @param {object} settings An object with settings from the store.
+ *
+ * @returns {array} An array of objects that can be used to render the PostSettingsModal.
+ */
+const modalContent = ( settings ) => [
+	{
+		title: __( "Google preview", "wordpress-seo" ),
+		content: <SnippetEditor hasPaperStyle={ false } />,
+		shouldRender: true,
+	},
+	{
+		title: __( "Facebook preview", "wordpress-seo" ),
+		content: <FacebookContainer />,
+		shouldRender: window.wpseoScriptData.metabox.showSocial.facebook,
+	},
+	{
+		title: __( "Twitter preview", "wordpress-seo" ),
+		content: <TwitterContainer />,
+		shouldRender: window.wpseoScriptData.metabox.showSocial.twitter,
+	},
+	{
+		title: __( "Schema", "wordpress-seo" ),
+		content: <SchemaTabContainer />,
+		shouldRender: settings.displaySchemaSettings,
+	},
+	{
+		title: __( "Advanced", "wordpress-seo" ),
+		content: <AdvancedSettings />,
+		shouldRender: settings.displayAdvancedTab,
+	},
+];
+
+/**
+ * Renders the Modal content.
+ *
+ * @param {object} props Children of the PostSettingsModal.
+ *
+ * @returns {*} Functional component that renders the content of the modal.
+ */
+const DrawerContainer = ( { children } ) => {
+	return (
+		<div className="yoast-content-container">
+			{
+				children.map( ( child, index ) => {
+					const isOpen = index === 0;
+
+					return (
+						 child.shouldRender && (
+							<Collapsible
+								key={ index }
+								initialIsOpen={ isOpen }
+								title={ child.title }
+								headingProps={ {
+									level: 2,
+									fontSize: "1rem",
+									fontWeight: "normal",
+									color: "#A4286A",
```

Comment:
This is the `$color_pink_dark` from the `@yoast/style-guide`, you should use that instead.

Reply:
This was a half-hearted attempt at changing the color of the collapsible heading.

In the end, product decided that the collapsibles should be grey for now.